common/flatpak-run.c

```diff
@@ -2941,6 +2942,24 @@ flatpak_run_setup_base_argv (FlatpakBwrap   *bwrap,
   else if (g_file_test ("/var/lib/dbus/machine-id", G_FILE_TEST_EXISTS))
     flatpak_bwrap_add_args (bwrap, "--ro-bind", "/var/lib/dbus/machine-id", "/etc/machine-id", NULL);
 
+  /* As per https://systemd.io/CONTAINER_INTERFACE parse os-release and pass through all
+   * key-value pairs with a service_host_ prefix added to the key */
+  if (g_file_get_contents ("/usr/lib/os-release", &os_release, NULL, NULL))
```

Comment:
My understanding is that it's better to open `/etc/os-release`, which is allowed to have local modifications, falling back to `/usr/lib/os-release` if `/etc/os-release` doesn't exist. (See `os-release(5)`)

Reply:
Good point, fixed